waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/metrics/MetastoreAvailabilityMetric.java

```diff
@@ -0,0 +1,18 @@
+package com.hotels.bdp.waggledance.metrics;
+
+public enum MetastoreAvailabilityMetric {
+
+  IS_AVAILABLE(0),
+  IS_UNAVAILABLE(1),
+  IS_UNKNOWN(2);
+
+  private final int availabilityValue;
```

Comment:
Can you move this to the `MetaStoreStatus` enum. Have a `getIntValue` method on it so we don't need two enums?

Reply:
Didn't realise this is our thing, I will absolutely do that. Nice catch 🙌🏻